doc/tutorials/snippet-sharing-rest-service.md

```diff
@@ -225,3 +225,129 @@ request that does not access a static file (js, css, png, md, etc.). If you omit
 this step, it will not be possible to use direct links to access resources of the
 frontend; accessing the frontend from the `index.html` will still work though.
 
+## Example Structure ##
+
+The following is a description of the setup we used for the Elektra website
+reachable at https://www.libelektra.org.
+
+### Web Server ###
+
+As web server we are using an Apache2 with the version coming with Debian Jessie.
+Several domains are used for different tasks, whereas only two are relevant for
+the here described service:
+
+- `restapi.libelektra.org` for the API provided by the `rest-backend`
+- `www.libelektra.org` for the website and frontend provided by the `rest-frontend`
+
+The server redirects requests on port 80 (non-SSL) to 443 using a very simple
+configuration like
+```
+# file: /etc/apache2/sites-available/www.libelektra.org.conf
+<VirtualHost *:80>
+    ServerName www.libelektra.org
+    Redirect permanent / https://www.libelektra.org/
+</VirtualHost>
+```
+for the `www.libelektra.org` domain (similar for `restapi.libelektra.org`).
+
+The secured variant of the configuration looks like
+```
+# file: /etc/apache2/sites-available/www.libelektra.org-le-ssl.conf
+<IfModule mod_ssl.c>
+<VirtualHost *:443>
+    ServerName www.libelektra.org
+
+    DocumentRoot "/usr/local/share/elektra/tool_data/rest-frontend/public"
```

Comment:
State (at the beginning) that we assume now that Elektra is installed to /usr/local.